reactor-test/src/test/java/reactor/test/DefaultContextExpectationsTest.java

```diff
@@ -292,7 +321,27 @@ public void notMatchesWithDescriptionAndScenarioName() {
 						base.matches(Objects::isNull, "someDescription")
 						    .then()
 						    .expectNextCount(10)::verifyComplete)
-				.withMessage("[scenario] Context Context1{foo=bar} doesn't match predicate someDescription");
+				.withMessageStartingWith("" +
+						"[scenario] Context doesn't match predicate someDescription\n" +
+						"Context: Context1{foo=bar}"
+				);
+	}
+
+	@Test
+	public void capturedOperator() {
+		assertContextExpectationFails(
+				s -> s.doOnEach(__ -> {}),
```

Comment:
would it make sense / demonstrate additional value to make the transformation / tested chain the same in both `capturedOperator` and `capturedOperatorWithDebug`?